openassessment/xblock/schema.py

```diff
@@ -63,6 +63,12 @@ def datetime_validator(value):
     u'student-training'
 ]
 
+VALID_UPLOAD_FILE_TYPES = [
```

Comment:
Nit: I find this to be a very strange set of values. I wonder if it wouldn't be clearer to change the ORA setting to be an array of valid file types.